framework/generated/generated_dx12_enum_to_string.cpp

```diff
@@ -32,141 +32,141 @@ GFXRECON_BEGIN_NAMESPACE(util)
 
 std::string ToString(const DXGI_RESIDENCY& value)
 {
+    const char* ret = "Unhandled DXGI_RESIDENCY";
     switch (value) {
-    case DXGI_RESIDENCY_FULLY_RESIDENT: return "DXGI_RESIDENCY_FULLY_RESIDENT";
-    case DXGI_RESIDENCY_RESIDENT_IN_SHARED_MEMORY: return "DXGI_RESIDENCY_RESIDENT_IN_SHARED_MEMORY";
-    case DXGI_RESIDENCY_EVICTED_TO_DISK: return "DXGI_RESIDENCY_EVICTED_TO_DISK";
-    default: break;
+        case DXGI_RESIDENCY_FULLY_RESIDENT: ret = "DXGI_RESIDENCY_FULLY_RESIDENT"; break;
+        case DXGI_RESIDENCY_RESIDENT_IN_SHARED_MEMORY: ret = "DXGI_RESIDENCY_RESIDENT_IN_SHARED_MEMORY"; break;
+        case DXGI_RESIDENCY_EVICTED_TO_DISK: ret = "DXGI_RESIDENCY_EVICTED_TO_DISK"; break;
```

Comment:
Mark, I'm seeing this for the first time now coincidentally while I was in the commit history looking for the commit which fixed #1349. I think I have lost a lot of your questions / comments that do not show up on the current front page of the issue, and I never noticed at the time. There was another one I half remember seeing about const something or other. Thank you so much for your feedback and I'm super sorry if I ignored stuff. Maybe I moved the github emails telling me about them to a folder enmasse with the notifications about ci builds working.

That warning/error is a great help: it tells us that we missed one of the enumerants. If we have cases for them all, no warning. The default suppresses that and so hides a bug. A default can be useful if it logs the error or something to handle it and it can detected a completely invalid input, say from memory corruption, but a do-nothing default is dangerous.

This commit was about the single return path, having one place where we call the code to build a string per function rather than many. I just stomped the default while I was in there.

Reply:
A lot of times it's hidden under something stat says "XX more comments hidden".  I hate that and wish Github had a mechanism for disabling the hiding, so I totally get it.

And thanks for responding.